composer/callbacks/memory_monitor.py

```diff
@@ -101,6 +141,14 @@ def after_train_batch(self, state: State, logger: Logger):
             return
 
         memory_report = _get_memory_report(self.memory_keys)
+        if self.dist_aggregate_batch_interval is not None and state.timestamp.batch.value % self.dist_aggregate_batch_interval == 0:
+            dist_memory_report = {}
+            for (mem_stat, val) in memory_report.items():
+                # only logged if dist_aggregate_batch_interval is set
```

Comment:
```suggestion
```
Sorry I wasn't clear. The docstring for this class describes what values are logged in a table. We should add a comment in the class description that additional things are conditionally logged (no need to add to table)

Reply:
[that](https://github.com/mosaicml/composer/pull/3042/commits/a2413de956a7670d75ff4eefcc8aaa385b7375bd) good